features/user_can_create_profile.feature

```diff
@@ -0,0 +1,59 @@
+@javascript
+Feature: User can create a Host Profile
+  As a registered user
+  In order to be able to apply for becoming a cat host
+  I need to create a host profile
+
+  Background:
+    Given the following user exist
+      | email        | password   |
+      | boa@snake.se | pswrd12345 |
+
+  Scenario: Registered user tries to create a Host Profile while not logged in (relatively happy path)
+    Given I visit the landing page
```

Comment:
Okay, this is good. But what if the user manually navigates to the page where he can create a new profile. For instance: `http://localhost:3000/profiles/new` ?

Reply:
Yes, we raised this with Faraz before and he told us not to worry about it just yet. But ok, we shall look at it then.

Reply:
Done.